projects/core/src/forms/control-label/control-label.element.scss

```diff
@@ -25,18 +25,20 @@
   text-transform: var(--text-transform) !important;
   cursor: pointer !important;
 
+  font-size: $cds-global-typography-secondary-font-size !important;
+  font-weight: $cds-global-typography-secondary-font-weight !important;
+  color: $cds-global-typography-color-400 !important;
+  line-height: $cds-global-typography-secondary-line-height !important;
+  letter-spacing: $cds-global-typography-secondary-letter-spacing !important;
+}
+
+::slotted([slot='label']:not(.cds-required-mark)) {
```

Comment:
I think that we should add `display: inline` in the `.cds-required-mark::before` styles and revert this change.